deployment/CP02PMCO_Deploy.csv

```diff
@@ -91,7 +100,7 @@ AR-31,,AR-34,,CP02PMCO-WFP01-04-FLORTK000,11,1,2018-11-02T20:28:00,2019-04-23T16
 AR-31,,AR-34,,CP02PMCO-WFP01-05-PARADK000,11,1,2018-11-02T20:28:00,2019-04-23T16:12:00,CGMCP-02PMCO-00011,CGVEH-CWFPCP-99102,CGINS-PARADK-20447,40.09675,-70.87898333,,127,148,
 AR-34,,AR-39,,CP02PMCO-RII01-02-ADCPTG010,12,1,2019-04-22T04:09:00,2019-10-08T12:02:00,CGMCP-02PMCO-00012,,CGINS-ADCPTG-19336,40.1012,-70.8877,,128,148,
 AR-34,,AR-39,,CP02PMCO-SBS01-00-STCENG000,12,1,2019-04-22T04:09:00,2019-10-08T12:02:00,CGMCP-02PMCO-00012,,CGCON-BSTC01-00010,40.1012,-70.8877,,0,148,
-AR-34,,AR-39,,CP02PMCO-SBS01-01-MOPAK0000,12,1,2019-04-22T04:09:00,2019-10-08T12:02:00,CGMCP-02PMCO-00012,,CGINS-MOPAK0-99999,40.1012,-70.8877,,0,148,sn not in config
+AR-34,,AR-39,,CP02PMCO-SBS01-01-MOPAK0000,12,1,2019-04-22T04:09:00,2019-10-08T12:02:00,CGMCP-02PMCO-00012,,CGINS-MOPAK0-99999,40.1012,-70.8877,,0,148,
 AR-34,,AR-39,,CP02PMCO-WFP01-00-WFPENG000,12,1,2019-04-22T04:09:00,2019-10-08T12:02:00,CGMCP-02PMCO-00012,CGVEH-CWFPCP-10305,CGCON-CWFPCP-10305,40.1012,-70.8877,,127,148,
 AR-34,,AR-39,,CP02PMCO-WFP01-01-VEL3DK000,12,1,2019-04-22T04:09:00,2019-10-08T12:02:00,CGMCP-02PMCO-00012,CGVEH-CWFPCP-10305,CGINS-VEL3DK-00034,40.1012,-70.8877,,127,148,
 AR-34,,AR-39,,CP02PMCO-WFP01-02-DOFSTK000,12,1,2019-04-22T04:09:00,2019-10-08T12:02:00,CGMCP-02PMCO-00012,CGVEH-CWFPCP-10305,CGINS-DOFSTK-02628,40.1012,-70.8877,,127,148,
```

Comment:
OK except as noted above, PMCO 4 water depths.

Reply:
Taken from cruise notebook (at my desk)

Reply:
Looks good to me